### ISSUE TITLE
A11Y - Keyboard navigation using Tab does not work

### ISSUE THREAD

**Issue by jisedlac** [oracle/visualvm#52]
Starting from VisualVM 1.3.9, keyboard navigation in application views (Summary, Profiler etc.) doesn't work.

**Comment by jisedlac** [comment]
Fixed in master.
